upgrade/Stage_0_Prerequisites.md

```diff
@@ -49,13 +49,33 @@ after a break, always be sure that a typescript is running before proceeding.
    - If `ncn-m001` has internet access, then use the following commands to download and install the latest documentation.
 
       > **Important:** The upgrade scripts expect the `docs-csm` RPM to be located at `/root/docs-csm-latest.noarch.rpm`; that is why these commands copy it there.
+      > ***NOTE:*** CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
```

Comment:
Might need an empty line between 51 and 52.  This ends up rendering without a line break before "**NOTE:**".